brittany/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM amazonlinux:2.0.20200722.0
+FROM ubuntu:18.04
```

Comment:
Oops, I shouldn't have done this. Reverted in 64830d767a4eb58aff4e1f1d72fcb489be367893.